tests/test_core.py

```diff
@@ -316,7 +316,7 @@ def test_list_objects1(self):
         # from send_secure_cmd
         session.send_secure_cmd.assert_called_with(
             72,
-            b"\x01\x00\x02\x02\x01\x03\xff\xff\x04\x00\x00\x7f\xff\xff\xff\xff\xff\x05\x15",  # noqa E501
+            b"\x01\x00\x02\x02\x01\x03\xff\xff\x04\x00?\xff\xff\xff\xff\xff\xff\x05\x15",  # noqa E501
```

Comment:
This change is due to additional CAPABILITY flags? For readability I would change the `?` to use the raw byte in `\xXX` format.